DEVELOPMENT.md

```diff
@@ -26,3 +26,15 @@ Publish the draft release
 If you're debugging a running server than `IO.inspect` is a good approach, any messages you create with it will be sent to your LSP client as a log message
 
 To debug in tests you can use `IO.inspect(Process.whereis(:user), message, label: "message")` to send your output directly to the group leader of the test process.
+
+# Documentation website
+
+The documentation website is built using the [Mkdocs](https://www.mkdocs.org) static website generator. The content is written in Markdown format in the directory [docs](./docs) and is configured via the [mkdocs.yml](./mkdocs.yml) file.
```

Comment:
Do you have a list of any additional packages needed? I've partially gotten this running locally and it looks like you need at least `pip install mkdocs-material`, are there any others? Can we add a note about them?

Reply:
Added the required packages in the doc.